src/ray/protobuf/experimental/instance_manager.proto

```diff
@@ -143,3 +145,71 @@ service InstanceManagerService {
   rpc GetAvailableInstanceTypes(GetAvailableInstanceTypesRequest)
       returns (GetAvailableInstanceTypesResponse);
 }
+
+// ============= Resource Scheduling Service API =======================
```

Comment:
per discussion: proto is not necessary unless we really need this service? a python interface should be sufficient.